Source/SwiftLintFramework/Rules/Metrics/LargeTupleRuleExamples.swift

```diff
@@ -0,0 +1,74 @@
+struct LargeTupleRuleExamples {
+    static var nonTriggeringExamples: [Example] {
+        let commonExamples = [
+            Example("let foo: (Int, Int)\n"),
+            Example("let foo: (start: Int, end: Int)\n"),
+            Example("let foo: (Int, (Int, String))\n"),
+            Example("func foo() -> (Int, Int)\n"),
+            Example("func foo() -> (Int, Int) {}\n"),
+            Example("func foo(bar: String) -> (Int, Int)\n"),
+            Example("func foo(bar: String) -> (Int, Int) {}\n"),
+            Example("func foo() throws -> (Int, Int)\n"),
+            Example("func foo() throws -> (Int, Int) {}\n"),
+            Example("let foo: (Int, Int, Int) -> Void\n"),
+            Example("let foo: (Int, Int, Int) throws -> Void\n"),
+            Example("func foo(bar: (Int, String, Float) -> Void)\n"),
+            Example("func foo(bar: (Int, String, Float) throws -> Void)\n"),
+            Example("var completionHandler: ((_ data: Data?, _ resp: URLResponse?, _ e: NSError?) -> Void)!\n"),
+            Example("func getDictionaryAndInt() -> (Dictionary<Int, String>, Int)?\n"),
+            Example("func getGenericTypeAndInt() -> (Type<Int, String, Float>, Int)?\n")
+        ]
+
+        guard SwiftVersion.current >= .fiveDotFive else {
+            return commonExamples
+        }
+
+        return commonExamples + [
```

Comment:
For both the triggering and non-triggering examples, I just copied the existing ones that involved functions and added `async`. Not all of these were actually false positives -- the issue is only with async closures -- but given that both `throws` and non-`throws` versions were tested for all of the functions already it seemed like getting coverage of the `async` versions would be a good idea too.